internal/controllers/operator_controller.go

```diff
@@ -272,6 +287,10 @@ func (r *OperatorReconciler) generateExpectedBundleDeployment(o operatorsv1alpha
 		"kind":       rukpakv1alpha1.BundleDeploymentKind,
 		"metadata": map[string]interface{}{
 			"name": o.GetName(),
+			"annotations": map[string]string{
+				"operator_version": annotations.version,
```

Comment:
The issue also mentions about adding the name of the operator that is being installed. That can be obtained from the operator's spec. If that's going to cause an additional client call, it can be added here.